source/_integrations/acaia.markdown

```diff
@@ -0,0 +1,103 @@
+---
+title: Acaia
+description: Instructions on how to integrate your Acaia smart coffee scale with Home Assistant.
+ha_release: 2024.12
+ha_category:
+  - Button
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_domain: acaia
+ha_platforms:
+  - button
+ha_bluetooth: true
+ha_codeowners:
+  - '@zweckj'
+ha_integration_type: device
+---
+
+The **Acaia** {% term integrations %} allows you to control [Acaia](https://acaia.co/) scales through Home Assistant.
+
+If your machine is within Bluetooth range to your Home Assistant host and the [Bluetooth](/integrations/bluetooth) integration is fully loaded, the scale should be discovered automatically.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+MAC:
+  description: "The MAC address of your scale."
+  required: true
+Is new style scale:
+  description: "Check this if your scale is a newer model (manufactured after 2021). This setting affects how commands are sent to the scale. If your scale doesn't respond to commands, try toggling this setting. For example, Lunar models made after 2021 should have this enabled."
+  required: false
```

Comment:
```suggestion
```
Still got it wrong. The "required" doesn't apply here either, as the UI will take care of that.